use/adopter/whatsapp-flow-for-uci.md

```diff
@@ -0,0 +1,218 @@
+# Whatsapp Flow (Netcore)
+
+## Types of Bots:
+
+1. **Broadcast Bot:** This type of bot is used to send a single message to a large group of people. It doesn't expect a reply to the message.
+
+2. **Conversation Bot:** This type of bot is designed for a proper conversation flow with a user. It facilitates a two-way conversation that is regulated by the ODK form.
```

Comment:
nit

- The first statement is unnecessary/lacks clarity. What is meant by a 'proper' conversation?
- regulated by 'an' ODK form.